ladder/teams/README.md

```diff
@@ -0,0 +1,67 @@
+# How Cilium GitHub Teams Are Managed
+The Cilium project makes use of GitHub teams to help enforce responsibility boundaries with regard to the project codebase here on GitHub.
+GitHub teams loosely map groups of [contributors](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md) to the Cilium Project as either reviewers and/or committers, focused on a portion of the codebase. [GitHub teams](https://github.com/orgs/cilium/teams) are an implementation detail in how code access and review responsibilities are organized and are not a 1 to 1 mapping to [Cilium Project teams](https://github.com/cilium/community/blob/main/CONTRIBUTOR-ROLES.md).
+
+For example, the Cilium Project Code team, may require multiple GitHub teams in the Cilium GitHub organization to self-organize their work. 
+These multiple GitHub teams will be composed of contributors at different steps along the contributor ladder: organizational members, reviewers, and committers.  
+The different GitHub teams have different permissions, corresponding to the level of code access needed by the members of that team. 
+
+Cilium GitHub organization team memberships can be self-managed by making use of team membership configuration files in this directory.
+
+Each Cilium GitHub team can establish a file in this directory managed by the CODEOWNERS system with the assistance of 
```

Comment:
```suggestion
Each Cilium GitHub team can establish a file in this directory managed by the CODEOWNERS system with the assistance of 
```

It's a bit odd to state that a team can create a file in this directory. The team doesn't exist until the file exists in this directory.

I wonder if this particular case would be better documented under a new section for creating and deleting teams. Then we could also expand on the expectations for these. This PR can help provide some insights into that: https://github.com/cilium/community/pull/182 .

Reply:
I meant to also state that creating / removing teams could be a separate addition to the doc. I think that this PR is already (a) documenting what exists + (b) introducing the mentor concept, so it would be easier to get the core wording in place first and then iterate on top for stuff like team changes (which is a much rarer case in practice)

Reply:
Actually not true. The team can exist.. in the private full team management configuration. 
So i'm envisioning initial team creation being still github org admin tasking, because you'll probavly need to touch the base team management config with potentially an initial group of members in the base config and then CODEOWNERS config for that team gets setup as part of that.

And then that team becomes self-service with the introduction of the file in this directory.

Reply:
I think it would simplify this doc if we only inform readers how to create a new team. The case where there's some existing team that maybe exists somewhere but isn't defined by this repo should be the exception and people involved in those teams should already be well aware of how that works (or at least communicating frequently with the core maintainers).